fractal_tasks_core/create_ome_zarr.py

```diff
@@ -230,23 +249,43 @@ def create_ome_zarr(
         # Identify all wells
         plate_prefix = dict_plate_prefixes[plate]
 
-        plate_image_iter = glob(
-            f"{in_path}/{plate_prefix}_*.{image_extension}"
+        patterns = [f"{plate_prefix}_*.{image_extension}"]
+        if image_glob_patterns:
+            patterns.extend(image_glob_patterns)
+        plate_images = glob_with_multiple_patterns(
+            folder=str(in_path), patterns=patterns
         )
 
         wells = [
-            parse_filename(os.path.basename(fn))["well"]
-            for fn in plate_image_iter
+            parse_filename(os.path.basename(fn))["well"] for fn in plate_images
         ]
         wells = sorted(list(set(wells)))
 
         # Verify that all wells have all channels
         for well in wells:
-            well_image_iter = glob(
-                f"{in_path}/{plate_prefix}_{well}*.{image_extension}"
+            patterns = [f"{plate_prefix}_{well}*.{image_extension}"]
```

Comment:
To make this check slightly more strict, it should be:
`f"{plate_prefix}_{well}_*.{image_extension}"` (always an `_` after the well)
If we need to make it even more strict, we can come up with a complex regex match for "T0001F0001L01A01Z01C01" (number of digits can vary, but always TFLAZC with numbers in between)

Reply:
Given that we also match with plate_prefix, adding the TFLAZC match should not be necessary here. But I would add the _ after well

Reply:
> Given that we also match with plate_prefix, adding the TFLAZC match should not be necessary here. But I would add the _ after well

Agreed